app/controllers/pages_controller.rb

```diff
@@ -46,4 +46,6 @@ def manifest
       format.json
     end
   end
+
+  def cookies; end
```

Comment:
Can you add a `respond_to` block like the other actions - otherwise we get missing template errors for urls like `/cookies.txt`. Once you've done that add a test to `spec/requests/disallowed_format_spec.rb` to make sure it works.